PWGHF/HFC/Tasks/taskCorrelationDplusHadrons.cxx

```diff
@@ -105,6 +108,13 @@ struct HfTaskCorrelationDplusHadrons {
   Configurable<bool> activateQA{"activateQA", false, "Flag to enable debug histogram"};
   Configurable<int> nTpcCrossedRaws{"nTpcCrossedRaws", 70, "Number of crossed TPC Rows"};
   Configurable<float> cutCollPosZMc{"cutCollPosZMc", 10., "max z-vertex position for collision acceptance"};
+  // CCDB configuration
+  Configurable<std::string> ccdbUrl{"ccdbUrl", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
+  Configurable<std::string> cfgAssociatedEffCCDBPath{"cfgAssociatedEffCCDBPath", "", "CCDB path for associated efficiency"};
+  Configurable<std::string> cfgPromptEffCCDBPath{"cfgPromptEffCCDBPath", "", "CCDB path for trigger efficiency"};
+  Configurable<std::string> cfgFDEffCCDBPath{"cfgFDEffCCDBPath", "", "CCDB path for trigger efficiency"};
+  Configurable<int64_t> timestampCCDB{"timestampCCDB", -1, "timestamp of the efficiency files used to query in CCDB"};
+  Configurable<long> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
```

Comment:
```suggestion
  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
```
Use lowerCamelCase for names of variables and use the same name for the JSON variable.